### PR TITLE
Update tokens, type, mixins, and utils.

### PR DESCRIPTION
**Pull request by bridgetburke**
- Updated type, weight, and type size tokens
- Added file for mixins
- Added utilities for weight
- Added base styles
- Added font-stack mixin. Created classes based on current documentation for typography (e.g.,  .m-page-heading-3xl, etc..). 

Made some changes to token names for brevity. Also, now mostly using custom-properties for styling currently.

### CODE REVIEW

**Comment by jonearley** [comment]
@bridgetburke did you see my inline review comments?

**Comment by bridgetburke** [comment]
No! @jonearley I didn't get a notification or see any comments.

File: packages/css/src/scss/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    }
+  }
+}
+
+.m {
+  @include font-modifiers;
+}
+
+/* Color */
+$color-map: (
```
[the remark this answers is not in the document]
Comment (bridgetburke) [reply packages/css/src/scss/_mixins.scss]: This was fixed.

**Comment by jonearley** [comment]
Oh, I didn't noticed my inline comments were pending and I had to submit them still. Sorry!

File: packages/css/src/tokens.json [hunk 2]
```diff
@@ -303,31 +303,108 @@
...
+    {
+      "theme": "maize",
+      "name": "type-base-size",
+      "category": "typography",
+      "value": "16px",
+      "type": "number"
+    },
+    {
+      "theme": "maize",
+      "name": "xxx-large",
```
[the remark this answers is not in the document]
Comment (bridgetburke) [reply packages/css/src/tokens.json]: I'm changing to 'text-'
We spoke about having just the size unit on Slack a few weeks ago but I would think we need the prefix for clarity.

File: packages/css/src/tokens.json [hunk 3]
```diff
@@ -303,31 +303,108 @@
...
+      "name": "semibold",
       "category": "typography",
-      "value": "400",
+      "value": "600",
       "type": "font"
     },
     {
       "theme": "maize",
-      "name": "font-weight-semibold",
+      "name": "line-height-text",
```
[the remark this answers is not in the document]
Comment (bridgetburke) [reply packages/css/src/tokens.json]: added `line-height-default` for now and for future discssion.

**Pull request merged**